scripts/irods/test/test_iquery.py

```diff
@@ -224,3 +225,62 @@ def test_genquery2_honors_group_permissions__issue_8259(self):
                     self.admin.run_icommand(['ichmod', '-M', '-r', 'own', self.user.username, self.user.session_collection])
                     self.admin.run_icommand(['iadmin', 'rmgroup', group])
                     self.admin.run_icommand(['iadmin', 'moduser', self.user.username, 'type', 'rodsuser'])
+
+    def test_genquery2_sql_generation_for_sql_distinct_keyword__issue_8261(self):
+        with self.subTest('GenQuery2 does not automatically insert DISTINCT keyword into SQL'):
```

Comment:
Why are these subTests rather than distinct tests?

Reply:
They started out as distinct tests but in order to prove correctness, they require additional ufs resources. It was easier to combine them rather than repeat resource management calls.

Reply:
Okay, it's fine as is. Just wanted to make sure.